### PR TITLE
Deprioritize ushort in completion

### LINKED ISSUE #44459: Top-level statement completion interferes with 'using' directives
**Version Used**: features/SimplePrograms

🔗 This issue is very similar to #43439. It should be fixed in a similar way, except the match priority should only be altered in the global statement context where a `using` directive is applicable.

📝 A disabled test for this behavior is added in #44402.

**Steps to Reproduce**:

1. Type `us` in an empty document
2. Press <kbd>Tab</kbd>

**Expected Behavior**:

```
using
```

**Actual Behavior**:

```
ushort
```

### PR DESCRIPTION
**Pull request by genlu**
Fix #44459

### CODE REVIEW

File: src/Features/CSharp/Portable/Completion/KeywordRecommenders/UShortKeywordRecommender.cs [hunk 1]
```diff
@@ -29,7 +34,7 @@ protected override bool IsValidContext(int position, CSharpSyntaxContext context
                 context.IsStatementContext ||
                 context.IsGlobalStatementContext ||
                 context.IsObjectCreationTypeContext ||
-                (context.IsGenericTypeArgumentContext && !context.TargetToken.Parent.HasAncestor<XmlCrefAttributeSyntax>()) ||
+                (context.IsGenericTypeArgumentContext && context.TargetToken.Parent?.HasAncestor<XmlCrefAttributeSyntax>() == false) ||
```
Comment (CyrusNajmabadi) [inline src/Features/CSharp/Portable/Completion/KeywordRecommenders/UShortKeywordRecommender.cs:37]:
```suggestion
                (context.IsGenericTypeArgumentContext && context.TargetToken.GetRequiredParent().HasAncestor<XmlCrefAttributeSyntax>()) ||
```

**Review by CyrusNajmabadi — APPROVED** [review]
Can you add an explicit test for this in CompletionCommandHandlerTests

**Comment by CyrusNajmabadi** [comment]
I sign off with a test added.  thanks.

**Comment by genlu** [comment]
@CyrusNajmabadi The tests are already added by @sharwell 
https://github.com/dotnet/roslyn/blob/664ee99951c40d7bb72ee5482b75efd5599fefb7/src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb#L267

**Comment by CyrusNajmabadi** [comment]
Yup. That works for me :)

**Pull request merged**